_posts/2020-11-09-pulling-most-recent-files.markdown

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title:  "Pull most recent files in R"
+date:   2020-11-09
+categories: jekyll update
+---
+# Pulling most recent files in R
+ ###### by Joel Christian Andrade
+
+Many coding pipelines involve taking raw data files, formulating code and eventually working with a few, if not many, output files.
+
+Take for example the list of files, in my directory `/raw_data` below from the publicly available Apple mobility database tracking movement.
```

Comment:
I get what you're trying to say here, but I think this sentence could be reworded to make it a little easier to read. Something like "Take for example this list of files from my `/raw_data` directory, which were downloaded from the publicly available Apple mobility database."

Reply:
reworded